Update robot server tests

Masters broken at the moment

```
--- FAIL: TestSetThermostat (9.01s)
    main_test.go:198: Expected request: angle:30 , got robotId:"qwerty" angle:30 
    main_test.go:198: Expected request: angle:100 , got robotId:"qwerty" angle:100 
    main_test.go:198: Expected request: angle:170 , got robotId:"qwerty" angle:170 
```

Need to fix this so we can merge #106.

--- a/thermostatserver/main_test.go
+++ b/thermostatserver/main_test.go
@@ -136,6 +136,7 @@ func TestSetThermostat(t *testing.T) {
 				Unit:       "kelvin",
 			},
 			expectedServoRequest: &robotserver.ServoRequest{
+				RobotId: "qwerty",
 				Angle: 30,
 			},
 		},
@@ -146,6 +147,7 @@ func TestSetThermostat(t *testing.T) {
 				Unit:       "kelvin",
 			},
 			expectedServoRequest: &robotserver.ServoRequest{
+				RobotId: "qwerty",
 				Angle: 100,
 			},
 		},
@@ -156,6 +158,7 @@ func TestSetThermostat(t *testing.T) {
 				Unit:       "kelvin",
 			},
 			expectedServoRequest: &robotserver.ServoRequest{
+				RobotId: "qwerty",
 				Angle: 170,
 			},
 		},

